HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -124,21 +128,40 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
 
     $("#submitbtn").prop('disabled', true);
 
+    // Separate SBATCH options from commands
+    job.commands = editor.getValue();
+    var other = editor.getValue().split("\n");
+    var sbatch = [];
+    var other = angular.forEach(other, function(line, index) {
+      if (line.includes("SBATCH")) {
+        this.push(line);
+        other.splice(index,1);
```

Comment:
Which `other` is this splicing?  There is the `var other` that you set this equal to, and the `var other` above that is the contents of the editor.